ci

```diff
@@ -506,18 +506,30 @@ case "$1" in
     run_log_commit "everest_move" "everest-upgrade" "#everest-build" "project-everest/everest"
     ;;
 
+  vale-ci)
+    # Run vale ci - start in everest
+    if ! [ -x everest ]; then
+      echo "Not in the right directory"
+      exit 1
+    fi
+
+    # Set up build environment
+    nuget restore vale/tools/Vale/src/packages.config -PackagesDirectory vale/tools/FsLexYacc 
+
+    run_log_commit "scons.bat -C vale -j 4" "vale-build" "#vale-build" "project-everest/everest/vale"
+    ;;
+
   *)
     cat <<USAGE
 USAGE: $0 ACTION
-
```

Comment:
it seems like you got rid of two blank lines that used to be there; is that intentional?